content/user/ui/settings/Archive/Groups.md

```diff
@@ -0,0 +1,74 @@
+<!--
+title: "Creating, Managing and Deleting Access Control Groups"
+description: "Creating, Managing and Deleting Access Control Groups"
+tags: "TeamServer groups acls"
+-->
+
+Access Control Groups provide a convenient way to manage a collection of users. Users can belong to many groups. Groups are used throughout TeamServer; for example, they can be used to:
+
+* Grant users system administration privileges.
+* Grant users read-only visibility to system administration information such as users, applications and groups for an organization.
+* Grant users access to organizations outside of their default organization. 
+* Grant users varying roles by organization.
+* Grant users varying roles by application. 
+* Restricted from accessing applications within an organization.
+
+## Viewing Groups
+TeamServer presently supports two types of groups: System and Organization. System groups only exist with EOP (Enterprise on Premise) installations. This type of group exists to extend system administration capabilities to named user accounts. Organization groups are available to both SaaS and EOP customers. This group exists for supporting application roles and restrictions. EOP customers may also use organization groups as a means of granting cross-organization access to a set of users.
+
+## Creating System Groups
+System groups are created from the system administration groups page by users given SuperAdmin credentials or users already granted as system administrators in a previously existing System group.
+
+<a href="assets/images/System_Role.png" rel="lightbox" title="System Groups"><img class="thumbnail" src="assets/images/System_Role.png"/></a>
+
+* Login to TeamServer.
+* Navigate to the Admin pages via the settings menu.
+* Select Groups from the top navigation menu.
+* Select "Create a Group".
+* Provide the required inputs.
+	* Group Name
+	* Type: Choose System
+	* Organization: You may choose all organizations or a single organization.
+	* Role: Observer, System Admin or No Access
+	* Group Members: Add one or more group members
+
+*Please Note*: A single group may contain multiple organizations by selecting "Add Role". Different roles can be applied to the same group as long as different roles are not assigned to the same organization within a group.
+
+## Creating Organization Groups (System Admin)
+EOP customers may setup organization groups that enable access and control across one or more organizations. Once a multi-organization group is configured, users can [navigate across organizations](user_tsguideset.html#toggle) from the settings menu. 
+
+<a href="assets/images/Multi_Org_Role_.png" rel="lightbox" title="Multi-Organization Group"><img class="thumbnail" src="assets/images/Multi_Org_Role_.png"/></a>
+
+* Login to TeamServer.
+* Navigate to the Admin pages via the settings menu.
```

Comment:
Admin pages, settings menu - can you do a quick sweep for consistency on how we are referencing touchpoints in the UI? I.e. using quotations vs. bold font vs. capitalization? In general, this bullet point doesn't make sense to me though - if you are logged in as an admin and then you are navigating to Groups from the top nav, it doesn't make sense why you would have to go to the Settings menu? I think we can take this step out.